src/main/kotlin/uk/gov/dluhc/emsintegrationapi/messaging/mapper/InitiateRegisterCheckMapper.kt

```diff
@@ -16,7 +17,8 @@ import java.util.UUID
 )
 interface InitiateRegisterCheckMapper {
 
-    @Mapping(target = "correlationId", source = "sourceCorrelationId")
+    // TODO EIP1-12676 Remove the source mapping and default expression here and replace both with an expression to assign a random UUID as part of the cleanup actions
+    @Mapping(target = "correlationId", source = "correlationId", defaultExpression = "java(UUID.randomUUID())")
```

Comment:
Nice, does this mean that `sourceCorrelationId` just gets passed though as-is from Register Checker, and we use the correlationId from reg check / create a new correlationId and return it from here